Lib/test/test_asyncio/test_subprocess.py

```diff
@@ -480,7 +480,9 @@ def test_popen_error(self):
             popen.side_effect = exc
 
             create = asyncio.create_subprocess_exec(sys.executable, '-c',
-                                                    'pass', loop=self.loop)
+                                                    'pass',
+                                                    stdin=subprocess.PIPE,
```

Comment:
Please add a new test instead of modifying the existing one.
Two tests for `stdin=PIPE` passed and `None` worth to be tested.
If you don't want to duplicate too much test code extract it into a separate method, like here: https://github.com/python/cpython/blob/master/Lib/test/test_asyncio/test_events.py#L572-L607